src/main/resources/assets/cbs/load_screens/cbs_mod_compat_pga.json

```diff
@@ -0,0 +1,9 @@
+{
+	"screens" : [{
+			"COMMENT" : "Custom Selection Box SUPPORT", 
+			"class" : "me.shedaniel.gui.CSBSettingsGUI", 
+			"texture" : "cbs:textures/gui/custom_selection_box.png", 
```

Comment:
The file and texture namespace should be csb?

Reply:
Yes. Forge itself looks for resources by default in assets/modid as far as i know.
This is to ensure it will work correctly in a forge environement  
Given the absence of a modid in fabric, i improvised it to cbs.

Reply:
So yeah I will be closing this as you should pr to the forge repo. 
https://gitlab.com/tennox-mods/CustomSelectionBox/

Reply:
on the 1.14 branch

Reply:
Also modid exists in fabric as well.

Reply:
Oh okay sorry i must have missed the forge fork. My bad !

I am not familiar with fabric and couldnt find anything that ressembled it. My bad.

Will move the pr when i can. Somewhere tonight probably.

Reply:
Thank you!